docs/querying/sql-functions.md

```diff
@@ -28,17 +28,39 @@ sidebar_label: "All functions"
  This document describes the SQL language.
 :::
 
+This page provides a reference of all Druid SQL functions in alphabetical order. The **Learn More** at the end of each function section provides further documentation.
 
-This page provides a reference of all Druid SQL functions in alphabetical order.
-Click the linked function type for documentation on a particular function.
+The examples for each function uses the following datasets that come included with Apache Druid:
+* `flight-carriers`: `FlightCarrierOnTime (1 month)` 
+* `trips_xaa`: `NYC Taxi cabs (3 files)`
 
 ## ABS
 
-`ABS(<NUMERIC>)`
+Calculates the absolute value of a numeric expression.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `ABS(<NUMERIC>)`
+* **Function type:** Scalar, numeric
 
-Calculates the absolute value of a numeric expression.
+<details><summary>Example</summary>
+
+The following applies the ABS function to the `ArrDelay` column, from the `flight-carriers` datasource.
+
+```sql
+SELECT
+  "ArrDelay",
```

Comment:
suggest aliasing column using `AS` to provide an English language column name. For example `arrival_delay`. Then you can use a similar English column name for the ABS: `absolute_arrival_delay`. This applies to all examples.
Also standardize all your column aliases to use lowercase and underscore.